packages/expo-av/ios/EXAV/EXAVPlayerData.m

```diff
@@ -156,7 +159,7 @@ - (void)_finishLoadingNewPlayer
           strongSelfInner.currentPosition = strongSelfInner.player.currentTime;
 
           if (strongSelfInner.shouldCorrectPitch) {
-            strongSelfInner.player.currentItem.audioTimePitchAlgorithm = AVAudioTimePitchAlgorithmLowQualityZeroLatency;
+            strongSelfInner.player.currentItem.audioTimePitchAlgorithm = strongSelfInner.pitchCorrectionQuality;
```

Comment:
probably we can get rid of this if-else and use `AVAudioTimePitchAlgorithmVarispeed` as a default value of `strongSelfInner.pitchCorrectionQuality`